praw/models/reddit/subreddit.py

```diff
@@ -1586,6 +1586,41 @@ def conversations(self, after=None, limit=None, other_subreddits=None,
             yield ModmailConversation.parse(data, self.subreddit._reddit,
                                             convert_objects=False)
 
+    def create(self, subject, body, recipient, author_hidden=False):
+        """Create a new modmail conversation.
+
+        :param subject: The message subject. Cannot be empty.
+        :param body: The message body. Cannot be empty.
+        :param recipient: The recipient; a username or an instance of
+            :class:`.Redditor`.
+        :param author_hidden: When True, author is hidden from non-moderators
+            (default: False).
+        :returns: A `.ModmailConversation` object for the newly created
+            conversation.
+
+        .. code:: python
+
+           subreddit = reddit.subreddit('redditdev')
+           redditor = reddit.redditor('bboe')
+           subreddit.modmail.create('Subject', 'Body', redditor)
+
+        """
+        for param in ['subject', 'body']:
+            argument = locals()[param]
+            if argument is None or argument == "":
+                raise ClientException("Message {param} cannot be empty."
```

Comment:
Is this check necessary? What does the API do when an empty string is provided?

I ask because I prefer simplicity on PRAW's side. If the server side raises an exception that is semi-intuitive for a user of PRAW to figure out what is needed, then I prefer that over doing client side checks. If however, the exception is misleading then a client side provides value.